release-doc-in-process/all_vf.TTL

```diff
@@ -30,6 +30,23 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
+vf:Recipe a               owl:Class ;
```

Comment:
Since we go with nested / overlapping processes to represent plans. We probably can in similar way use `vf:RecipeProcess` with similar nesting / overlapping. I understood that from Recipe after scaling quantities one would get a Plan.

Reply:
Are you thinking we don't need a Recipe class?  I put it there as a simple way to handle forks and versions.  And just different ways of producing the same thing. I think a data structure that doesn't include everything for a recipe will be just hard to deal with, especially with distributed information across lots of recipe creators.

Reply:
I think we should make it consistent wit how we model Plan
if from Recipe we get Plan
and we represent Plan as Process
we should represent Recipe as RecipeProcess
It just needs to map clearly to how we do things on Planning layer.

Reply:
Interesting thought.  Recipe and Plan are not always the same granularity.  Often plans will include more than one deliverable, which often implies more than one recipe - when the deliverables are for one purpose or goal, but produced separately.  Even if a recipe has more than one output, it would never include value flows that don't intersect, it is all connected processes.

But even with that, your point is still valid, still thinking about the idea.  Recipe does have some different properties too than you would find on a process type, but if we called out the purposes of processes and purposes of  process types, say with subclassing, then we could do that easily.

Reply:
I think we need to clarify how Plan and Recipe relate. I would expect that for any Plan we can create Recipe that would automate crating similar (isomorphic?) plans.

Reply:
In our experience: You can use one or more recipes to create a plan.  There's a bunch of logic that goes in between those though, not a data relationship that seems useful to me.  And often there are tweaks to the plan after generating from a recipe.

For any plan you could create a recipe for each deliverable, yes.  Which would be valid if the plan didn't rely on inventory, etc.  We have had requests to be able to create a recipe from a flow of actual processes and economic events, like "hey this worked pretty well, let's create a recipe".

Reply:
Maybe let's first figure out how we model plans and nested/overlapping processes on Planning layer and then we try to see what we need on Recipe layer to automate crating constructs on Planning layer? I think we can already start with `vf:ProcessRecipe` and `vf:FlowRecipe` and see how far this gets us.